rs/nns/governance/canbench/canbench_results.yml

```diff
@@ -1,153 +1,153 @@
 benches:
   add_neuron_active_maximum_stable:
     total:
-      instructions: 60061051
+      instructions: 61309494
```

Comment:
Do we have any idea what's driving the (in most cases slight) performance degradation with this rust upgrade?  Is there any subsequent tuning that could be done?

Reply:
I don't know. We could try running [wasm-opt](https://github.com/WebAssembly/binaryen?tab=readme-ov-file#wasm-opt) on our WASMs to see if it can speed them up.

Reply:
I suppose this tuning / improvement can be done in a separate PR, since it's not clear how exactly to improve performance. Since Max already approved the PR I presume he's okay with merging it and following up on the performance separately.